app/models/item.rb

```diff
@@ -2,4 +2,9 @@ class Item < ApplicationRecord
   belongs_to :merchant
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
+
+  def current_price
+    price = (unit_price * 0.001).round(2)
```

Comment:
You're rounding by one too many 0's in your decimal. If the unit price is in cents, we should multiply by `0.01` to get the price in dollars, not by `0.00`=1`.

Reply:
If you look in the Mod 2 cheat sheet bookmarked in the slack channel, there is actually a method specifically for converting to dolla dolla bills y'all!!

Reply:
I fixed it, I tried the method in the cheat sheet and it gave me thousands :/